feature/gribi/ate_tests/backup_nhg_multiple_nh_test/backup_nhg_multiple_nh_test.go

```diff
@@ -262,45 +283,49 @@ func testIPv4BackUpSwitch(ctx context.Context, t *testing.T, args *testArgs) {
 
```

Comment:
Can you help also update the comment of this func so that it's more aligned with the README? Thanks.

Reply:
done